### ISSUE TITLE
Add help text at Question set : ua_ruhr_v1.1.0-beta.1/content-classification/data-dataset

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#52]
**Describe the request**  
General: the term data set is not perceived as very clear by storage colleagues and could possibly be interpreted differently by scientists. Maybe the term data type, data group, data category should be used?

**Propose what exactly should be changed**  
Since the term record is frequently used in RDMO, a addition to the help text would be helpful.

**Proposed setting or content**   
Just a suggestion to complete the help text: 
"Als Datensätze können unterschiedliche Datenarten, Datengruppen oder Datenkategorien zusammengefasst beschrieben werden."

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (en and ger)

**Comment by jmfrenzel** [comment]
Changes to a question set would be a manual change since out catalog auto creation tool at the moment only supports questions and their content.

An option would be to make a PR upstream at https://github.com/rdmorganiser/rdmo-catalog

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/78

**Comment by jmfrenzel** [comment]
Die diskutierte erweiterte Hilfe https://github.com/rdmorganiser/rdmo-catalog/pull/84/commits/291ccc3a65bc7ce0fb87f0db370a6b359d3cbc3c

**Comment by ClaraRosalie** [comment]
implemented in upstream rdmorganiser#84
